[2024-08-24T10:51:39.588+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-24T10:51:39.601+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2024-08-24T03:30:04.316113+00:00 [queued]>
[2024-08-24T10:51:39.608+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2024-08-24T03:30:04.316113+00:00 [queued]>
[2024-08-24T10:51:39.608+0000] {taskinstance.py:2856} INFO - Starting attempt 6 of 6
[2024-08-24T10:51:39.621+0000] {taskinstance.py:2879} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2024-08-24 03:30:04.316113+00:00
[2024-08-24T10:51:39.625+0000] {standard_task_runner.py:72} INFO - Started process 206 to run task
[2024-08-24T10:51:39.628+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'spark_***_dag', 'spark_submit_task', 'manual__2024-08-24T03:30:04.316113+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/spark_jobs_dag.py', '--cfg-path', '/tmp/tmpgu2huuwy']
[2024-08-24T10:51:39.629+0000] {standard_task_runner.py:105} INFO - Job 53: Subtask spark_submit_task
[2024-08-24T10:51:39.641+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:209: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-24T10:51:39.668+0000] {task_command.py:467} INFO - Running <TaskInstance: spark_airflow_dag.spark_submit_task manual__2024-08-24T03:30:04.316113+00:00 [running]> on host 84408b420b45
[2024-08-24T10:51:39.731+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_***_dag' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-24T03:30:04.316113+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-24T03:30:04.316113+00:00'
[2024-08-24T10:51:39.732+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-08-24T10:51:39.732+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-08-24T10:51:39.733+0000] {logging_mixin.py:190} INFO - Current task name:spark_submit_task state:running start_date:2024-08-24 10:51:39.602047+00:00
[2024-08-24T10:51:39.733+0000] {logging_mixin.py:190} INFO - Dag name:spark_***_dag and current dag run status:running
[2024-08-24T10:51:39.733+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-24T10:51:39.741+0000] {base.py:84} INFO - Retrieving connection 'spark_local'
[2024-08-24T10:51:39.742+0000] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master spark://b53b649e194f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py
[2024-08-24T10:51:42.272+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkContext: Running Spark version 3.5.2
[2024-08-24T10:51:42.273+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-08-24T10:51:42.274+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkContext: Java version 17.0.12
[2024-08-24T10:51:42.316+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-08-24T10:51:42.384+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO ResourceUtils: ==============================================================
[2024-08-24T10:51:42.385+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-08-24T10:51:42.385+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO ResourceUtils: ==============================================================
[2024-08-24T10:51:42.385+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkContext: Submitted application: DataExtraction
[2024-08-24T10:51:42.401+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-08-24T10:51:42.409+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO ResourceProfile: Limiting resource is cpu
[2024-08-24T10:51:42.409+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-08-24T10:51:42.444+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SecurityManager: Changing view acls to: default
[2024-08-24T10:51:42.444+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SecurityManager: Changing modify acls to: default
[2024-08-24T10:51:42.445+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SecurityManager: Changing view acls groups to:
[2024-08-24T10:51:42.445+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SecurityManager: Changing modify acls groups to:
[2024-08-24T10:51:42.446+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: default; groups with view permissions: EMPTY; users with modify permissions: default; groups with modify permissions: EMPTY
[2024-08-24T10:51:42.610+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO Utils: Successfully started service 'sparkDriver' on port 42713.
[2024-08-24T10:51:42.630+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkEnv: Registering MapOutputTracker
[2024-08-24T10:51:42.653+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkEnv: Registering BlockManagerMaster
[2024-08-24T10:51:42.666+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-08-24T10:51:42.667+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-08-24T10:51:42.670+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-08-24T10:51:42.686+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-badfb1f1-cccd-4353-8832-478f5dbd5d08
[2024-08-24T10:51:42.696+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-08-24T10:51:42.706+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-08-24T10:51:42.795+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-08-24T10:51:42.859+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-08-24T10:51:42.971+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:42 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://b53b649e194f:7077...
[2024-08-24T10:51:43.023+0000] {spark_submit.py:579} INFO - 24/08/24 10:51:43 INFO TransportClientFactory: Successfully created connection to b53b649e194f/172.19.0.5:7077 after 29 ms (0 ms spent in bootstraps)
[2024-08-24T10:52:02.974+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:02 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://b53b649e194f:7077...
[2024-08-24T10:52:22.974+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:22 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://b53b649e194f:7077...
[2024-08-24T10:52:42.975+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2024-08-24T10:52:42.975+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2024-08-24T10:52:42.984+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 35357.
[2024-08-24T10:52:42.984+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO NettyBlockTransferService: Server created on 84408b420b45:35357
[2024-08-24T10:52:42.986+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-08-24T10:52:42.992+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 84408b420b45, 35357, None)
[2024-08-24T10:52:42.995+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO BlockManagerMasterEndpoint: Registering block manager 84408b420b45:35357 with 434.4 MiB RAM, BlockManagerId(driver, 84408b420b45, 35357, None)
[2024-08-24T10:52:42.997+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 84408b420b45, 35357, None)
[2024-08-24T10:52:42.998+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:42 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 84408b420b45, 35357, None)
[2024-08-24T10:52:43.189+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-08-24T10:52:43.377+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-08-24T10:52:43.389+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO SparkUI: Stopped Spark web UI at http://84408b420b45:4040
[2024-08-24T10:52:43.392+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-08-24T10:52:43.394+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-08-24T10:52:43.398+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2024-08-24T10:52:43.405+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-08-24T10:52:43.417+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO MemoryStore: MemoryStore cleared
[2024-08-24T10:52:43.417+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO BlockManager: BlockManager stopped
[2024-08-24T10:52:43.421+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-08-24T10:52:43.423+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-08-24T10:52:43.432+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:43 INFO SparkContext: Successfully stopped SparkContext
[2024-08-24T10:52:44.295+0000] {spark_submit.py:579} INFO - Traceback (most recent call last):
[2024-08-24T10:52:44.295+0000] {spark_submit.py:579} INFO - File "/opt/***/dags/spark_etl_script_docker.py", line 25, in <module>
[2024-08-24T10:52:44.297+0000] {spark_submit.py:579} INFO - RDD = sparkContext.parallelize([data])
[2024-08-24T10:52:44.297+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 783, in parallelize
[2024-08-24T10:52:44.298+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 630, in defaultParallelism
[2024-08-24T10:52:44.298+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-08-24T10:52:44.299+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-08-24T10:52:44.299+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.8/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-08-24T10:52:44.300+0000] {spark_submit.py:579} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o24.defaultParallelism.
[2024-08-24T10:52:44.300+0000] {spark_submit.py:579} INFO - : java.lang.IllegalStateException: Cannot call methods on a stopped SparkContext.
[2024-08-24T10:52:44.301+0000] {spark_submit.py:579} INFO - This stopped SparkContext was created at:
[2024-08-24T10:52:44.301+0000] {spark_submit.py:579} INFO - 
[2024-08-24T10:52:44.301+0000] {spark_submit.py:579} INFO - org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-24T10:52:44.301+0000] {spark_submit.py:579} INFO - java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-24T10:52:44.302+0000] {spark_submit.py:579} INFO - java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-08-24T10:52:44.302+0000] {spark_submit.py:579} INFO - java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-24T10:52:44.302+0000] {spark_submit.py:579} INFO - java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2024-08-24T10:52:44.302+0000] {spark_submit.py:579} INFO - java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2024-08-24T10:52:44.302+0000] {spark_submit.py:579} INFO - py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-24T10:52:44.303+0000] {spark_submit.py:579} INFO - py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-24T10:52:44.303+0000] {spark_submit.py:579} INFO - py4j.Gateway.invoke(Gateway.java:238)
[2024-08-24T10:52:44.303+0000] {spark_submit.py:579} INFO - py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-24T10:52:44.303+0000] {spark_submit.py:579} INFO - py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-24T10:52:44.304+0000] {spark_submit.py:579} INFO - py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-24T10:52:44.304+0000] {spark_submit.py:579} INFO - py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-24T10:52:44.304+0000] {spark_submit.py:579} INFO - java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T10:52:44.304+0000] {spark_submit.py:579} INFO - 
[2024-08-24T10:52:44.305+0000] {spark_submit.py:579} INFO - The currently active SparkContext was created at:
[2024-08-24T10:52:44.305+0000] {spark_submit.py:579} INFO - 
[2024-08-24T10:52:44.305+0000] {spark_submit.py:579} INFO - (No active SparkContext.)
[2024-08-24T10:52:44.305+0000] {spark_submit.py:579} INFO - 
[2024-08-24T10:52:44.306+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.assertNotStopped(SparkContext.scala:122)
[2024-08-24T10:52:44.306+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.defaultParallelism(SparkContext.scala:2702)
[2024-08-24T10:52:44.306+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-08-24T10:52:44.306+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-08-24T10:52:44.307+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-08-24T10:52:44.307+0000] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2024-08-24T10:52:44.307+0000] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-08-24T10:52:44.307+0000] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-24T10:52:44.308+0000] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-08-24T10:52:44.308+0000] {spark_submit.py:579} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-08-24T10:52:44.308+0000] {spark_submit.py:579} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-08-24T10:52:44.308+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-24T10:52:44.308+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-24T10:52:44.309+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T10:52:44.309+0000] {spark_submit.py:579} INFO - 
[2024-08-24T10:52:44.361+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:44 INFO ShutdownHookManager: Shutdown hook called
[2024-08-24T10:52:44.362+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:44 INFO ShutdownHookManager: Deleting directory /tmp/spark-b6faeb08-f7db-417b-b200-4ec932d4d51a/pyspark-85aaed65-531f-48bb-b256-cfbfd7d5966e
[2024-08-24T10:52:44.365+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:44 INFO ShutdownHookManager: Deleting directory /tmp/spark-9916fa47-0dbf-490e-b0a0-49b55ffb8674
[2024-08-24T10:52:44.368+0000] {spark_submit.py:579} INFO - 24/08/24 10:52:44 INFO ShutdownHookManager: Deleting directory /tmp/spark-b6faeb08-f7db-417b-b200-4ec932d4d51a
[2024-08-24T10:52:44.412+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://b53b649e194f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.
[2024-08-24T10:52:44.415+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-08-24T10:52:44.415+0000] {logging_mixin.py:190} INFO - Task start:2024-08-24 10:51:39.602047+00:00 end:2024-08-24 10:52:44.415063+00:00 duration:64.813016
[2024-08-24T10:52:44.416+0000] {logging_mixin.py:190} INFO - Task:<Task(SparkSubmitOperator): spark_submit_task> dag:<DAG: spark_***_dag> dagrun:<DagRun spark_***_dag @ 2024-08-24 03:30:04.316113+00:00: manual__2024-08-24T03:30:04.316113+00:00, state:running, queued_at: 2024-08-24 10:51:38.196842+00:00. externally triggered: True>
[2024-08-24T10:52:44.416+0000] {logging_mixin.py:190} INFO - Failure caused by Cannot execute: spark-submit --master spark://b53b649e194f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.
[2024-08-24T10:52:44.416+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=spark_***_dag, task_id=spark_submit_task, run_id=manual__2024-08-24T03:30:04.316113+00:00, execution_date=20240824T033004, start_date=20240824T105139, end_date=20240824T105244
[2024-08-24T10:52:44.431+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-24T10:52:44.432+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 53 for task spark_submit_task (Cannot execute: spark-submit --master spark://b53b649e194f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.; 206)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://b53b649e194f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.
[2024-08-24T10:52:44.474+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2024-08-24T10:52:44.488+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-24T10:52:44.491+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
